src/pages/Facility/settings/organizations/FacilityOrganizationUsers.tsx

```diff
@@ -50,121 +70,131 @@ export default function FacilityOrganizationUsers({ id, facilityId }: Props) {
+                          <div className="flex flex-col gap-1">
+                            <div className="flex items-start justify-between">
+                              <h1 className="text-base font-bold break-words pr-2">
+                                {userRole.user.first_name}{" "}
+                                {userRole.user.last_name}
+                              </h1>
+                              <span className="text-sm text-gray-500">
+                                <UserStatusIndicator user={userRole.user} />
+                              </span>
+                            </div>
+                            <span className="text-sm text-gray-500 mr-2 break-words">
                               {userRole.user.username}
                             </span>
-                            <UserStatusIndicator user={userRole.user} />
-                          </span>
+                          </div>
+                          <div className="mt-4 -ml-12 sm:ml-0 grid grid-cols-2 gap-2 text-sm">
+                            <div>
+                              <div className="text-gray-500">{t("role")}</div>
+                              <div className="font-medium truncate">
+                                {userRole.role.name}
+                              </div>
+                            </div>
+                            <div>
+                              <div className="text-gray-500">
+                                {t("phone_number")}
+                              </div>
+                              <div className="font-medium truncate">
+                                {userRole.user.phone_number}
+                              </div>
+                            </div>
+                          </div>
                         </div>
                       </div>
-                    </div>
 
-                    <div className="grid grid-cols-2 gap-4 text-sm">
-                      <div>
-                        <div className="text-gray-500">{t("role")}</div>
-                        <div className="font-medium truncate">
-                          {userRole.role.name}
-                        </div>
-                      </div>
-                      <div>
-                        <div className="text-gray-500">{t("phone_number")}</div>
-                        <div className="font-medium truncate">
-                          {userRole.user.phone_number}
-                        </div>
+                      <div className="mt-2 -mx-2 -mb-2 sm:-mx-4 sm:-mb-4 rounded-md py-4 px-4 bg-gray-50 flex justify-end">
+                        <EditFacilityUserRoleSheet
+                          facilityId={facilityId}
+                          organizationId={id}
+                          userRole={userRole}
+                          trigger={
+                            <Button variant="outline" size="sm">
+                              <span>{t("see_details")}</span>
```

Comment:
Hmm, imo we should update this to redirect to user's profile page and add an edit button, but that can be done in a different issue.